tests/accelerators/test_accelerator_connector.py

```diff
@@ -634,5 +634,5 @@ def test_accelerator_ddp_for_cpu(tmpdir):
 @pytest.mark.parametrize("precision", [1, 12, "invalid"])
 def test_validate_precision_type(tmpdir, precision):
 
-    with pytest.raises(MisconfigurationException, match=f"Precision {precision} is invalid"):
+    with pytest.raises(MisconfigurationException, match=f"Precision {repr(precision)} is invalid"):
```

Comment:
Using repr so it prints

```Precision 'invalid' is invalid```

instead of

```Precision invalid is invalid```

when a string is passed

Reply:
but I think precision can be int or str here. mostly int when it's set to 16, 32 or 64

Reply:
what do you mean? the behaviour is the same for ints

Reply:
oh my bad! I misunderstood this.